### ISSUE TITLE
Add "Runes Crafted" Overlay for Runecraft Plugin

### ISSUE THREAD

**Issue by OliverBroadbent** [runelite/runelite#9570]
I always find myself wanting to know how many runes i have crafted during my session and there's no easy way to do that unless you empty the rune you're crafting in your bank

**Describe the solution you'd like**
Just like the mining and woodcutting plugins, display the amount of runes crafted via on overlay in the Runecraft plugin.

**Comment by Alexsuperfly** [comment]
duplicate of #4126

**Comment by OliverBroadbent** [comment]
> duplicate of #4126

Not really a duplicate, but okay.

**Comment by Alexsuperfly** [comment]
so you dont want to track runes crafted in a session?

**Comment by OliverBroadbent** [comment]
> so you dont want to track runes crafted in a session?

I'm not suggesting a whole new plugin be created to track all resources gathered. I'm suggesting, like the fishing and woodcutting plugins, display the amount of runes crafted. It would be implemented the exact same way.

Yes, I guess it would fall under the same criteria as #4126, but that's not what i'm suggesting.

**Comment by dekvall** [comment]
An implementation of #4126 would solve the issue at hand, though, he is specifically asking for an overlay here. Not an extension of the loottracker. It would be nice if all of these overlays (wc, fish etc) were moved to a plugin of the sort.

**Comment by OliverBroadbent** [comment]
> An implementation of #4126 would solve the issue at hand, though, he is specifically asking for an overlay here. Not an extension of the loottracker. It would be nice if all of these overlays (wc, fish etc) were moved to a plugin of the sort.

Agreed. I'm not objecting to #4126, bc i actually really like the idea.

**Comment by MRG95** [comment]
I don't believe this is a duplicate. An overlay for this specific thing is different than the other proposed suggestion because the other suggestion includes more things in a different format and in a completely new plugin.

This issue serves as an addition to an existing plugin and in a different format.
